cli/src/generate/prepare_grammar/extract_tokens.rs

```diff
@@ -51,12 +51,18 @@ pub(super) fn extract_tokens(
             index,
         }) = variable.rule
         {
-            if i > 0 && extractor.extracted_usage_counts[index] == 1 {
+            if i > 0 && extractor.extracted_usage_counts[index] == 1
+            //&& variable.kind != VariableType::Hidden
```

Comment:
🔥 ?

Reply:
whoops, extraneous comment I didn't mean to add